Dockerfile.devcontainer

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/vscode/devcontainers/go:0-1.19
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Uninstall pre-installed formatting and linting tools
+# They would conflict with our pinned versions
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    git \
+    make \
```

Comment:
make and git are already installed with the base image

Reply:
please add vim as by default only vi is installed

Reply:
What's the reason you want to have Vim added? IMO this is personal preference, although I might be missing something. I myself mainly use nano and I know people who prefer emacs.

Reply:
because the default commit message editor in devcontainers is vim and it falls back to vi, if vim is not installed.

Reply:
Done